dotnet/samples/Demos/A2AClientServer/A2AClient/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Reflection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+
+namespace A2A;
+
+public static class Program
+{
+    public static async Task<int> Main(string[] args)
+    {
+        // Create root command with options
+        var rootCommand = new RootCommand("A2AClient");
+        rootCommand.SetHandler(HandleCommandsAsync);
+
+        // Run the command
+        return await rootCommand.InvokeAsync(args);
+    }
+
+    public static async System.Threading.Tasks.Task HandleCommandsAsync(InvocationContext context)
+    {
+        await RunCliAsync();
+    }
```

Comment:
There are no arguments setup for the project this can be simplified and actually used in the Server impl.

OR just use the full RunCliAsync code in the `Main`.


```suggestion
    public static Task Main(string[] args)
    {
        return RunCliAsync();
    }
```